dex-it/src/test/scala/com/wavesplatform/it/sync/orders/OrderDynamicFeeTestSuite.scala

```diff
@@ -645,6 +645,8 @@ class OrderFeeTestSuite extends MatcherSuiteBase {
   }
 
   "rounding fee to filled amount" - {
+    val matcherFee = 300000
```

Comment:
use the global fee constant (from package or base suite)

Reply:
ok